Sources/SwiftQueue/SwiftQueueManager.swift

```diff
@@ -118,7 +118,25 @@ public final class SwiftQueueManager {
         }
         return count
     }
+}
+
+internal extension SwiftQueueManager {
+
+    func getAllAllowBackgroundOperation() -> [SqOperation] {
+        return manage.values
```

Comment:
In general, when chaining filter/map operations, I recommend to use .lazy, that I will bring some performance gain.

Reply:
Thanks. I'll check on the library all the collection I have that can be made lazy